robot/control/Inc/modules/BatteryModule.hpp

```diff
@@ -6,19 +6,44 @@
 
 #include "drivers/Battery.hpp"
 
+/**
+ * Module interfacing with battery and handling battery status
+ */
 class BatteryModule : public GenericModule {
 public:
-    // How many times per second this module should run
-    static constexpr float kFrequency = 1.0f; // Hz
+    /**
+     * Number of times per second (frequency) that BatteryModule should run (Hz).
+     */
+    static constexpr float kFrequency = 1.0f;
+
+    /**
+     * Number of seconds elapsed (period) between BatteryModule runs (milliseconds)
+     */
     static constexpr std::chrono::milliseconds kPeriod{static_cast<int>(1000 / kFrequency)};
+
+    /**
+     * Priority used by RTOS
+     *
+     * Uses the default priority of 1.
+     */
     static constexpr int kPriority = 1;
 
+    /**
+     * Constructor for BatteryModule
+     *
+     * @param batteryVoltage Packet of data containing data on battery voltage and critical status
+     */
     explicit BatteryModule(LockedStruct<BatteryVoltage>& batteryVoltage);
 
+    /**
+     * Code to run when called by RTOS
```

Comment:
This is applicable to all module comments

Reply:
I'm debating using the copy docs and inherit functionality, because my rationale is that it would be easier to just see the comments right in front of your eyes when you are looking through a file. Although, I suppose you might make the case that one can just take a look at the parent class

Reply:
Yeahh, it's could go either way. Honestly, pick a version and stick with it and everyone else will follow suite. I just wanted to make sure you knew about those options